cmd/harmony/main.go

```diff
@@ -217,6 +219,11 @@ func setUpConsensusAndNode(nodeConfig *nodeconfig.ConfigType) (*consensus.Consen
 	// TODO: consensus object shouldn't start here
 	// TODO(minhdoan): During refactoring, found out that the peers list is actually empty. Need to clean up the logic of consensus later.
 	currentConsensus, err := consensus.New(nodeConfig.Host, nodeConfig.ShardID, nodeConfig.Leader, nodeConfig.ConsensusPriKey)
+	attackInstance := attack.GetInstance()
+	if *delayAttack {
+		attackInstance.SetDelayResponseAttack()
```

Comment:
no need to create an instance - just use attack.GetInstance().SetDelayResponseAttack()